/**:
  ros__parameters:
    cam_width:  640.0
    cam_height: 480.0
    cam_fx: 346.74048
    cam_fy: 349.12708
    cam_cx: 322.83899
    cam_cy: 234.54992

    cam2body_R: [ 0.0,  0.0,  1.0,
                 -1.0,  0.0,  0.0,
                  0.0, -1.0,  0.0]
    cam2body_p: [0.0, 0.0, 0.1]

